src/components/Keyboard/SwitchKeyboard/SwitchKeyboard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './SwitchKeyboard.css';
+import { QWERTY, ALPHABETICALLY } from '../../../../consts';
+import Alphabet from '../Alphabet/Alphabet';
+
+// eslint-disable-next-line max-len
+const allLetters = ['A', 'Ą', 'B', 'C', 'Ć', 'D', 'E', 'Ę', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'Ł', 'M', 'N', 'Ń', 'O', 'Ó', 'P', 'Q', 'R', 'S', 'Ś', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'Ź', 'Ż'];
+const qwertyLettersPart1 = ['Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P'];
+const qwertyLettersPart2 = ['A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L'];
+const qwertyLettersPart3 = ['Z', 'X', 'C', 'V', 'B', 'N', 'M'];
+const latinLetters = ['Ą', 'Ć', 'Ę', 'Ł', 'Ń', 'Ó', 'Ś', 'Ź', 'Ż'];
+
+class SwitchKeyboard extends React.PureComponent {
+    state = {
+        choosenKeyboard: localStorage.getItem('choosenKeyboard') || QWERTY,
```

Comment:
`localStorage.getItem('choosenKeyboard')` - use instead of `'choosenKeyboard'` some constant

Reply:
I don't understand, it must be a string.